includes/base/widget-base.php

```diff
@@ -254,12 +256,10 @@ public function hide_on_search() {
 	public function start_controls_section( $section_id, array $args = [] ) {
 		parent::start_controls_section( $section_id, $args );
 
-		static $is_first_section = true;
-
-		if ( $is_first_section ) {
+		if ( $this->is_first_section ) {
```

Comment:
I think it cannot be as private property because it should be available for each widget render to avoid an infinite loop. please talk to me about it.